models/proxy.go

```diff
@@ -70,6 +76,13 @@ func ParseProxyConfigFromFile(cfgFile string) (*Proxy, error) {
 	if proxyConfig.ConfigType == "" {
 		proxyConfig.ConfigType = ConfigEtcd
 	}
+	if proxyConfig.Cluster == "" && proxyConfig.CoordinatorRoot == "" {
```

Comment:
Which field is using when both `cluster_name` and `coordinator_root` are set?

Reply:
we use cluster_name when both cluster_name and coordinator_root are set